README.md

```diff
@@ -176,37 +176,56 @@ When the issue is created, it is required to fill in the following information:
 3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
    under "Proposed by".
 
-### 2. Identify roles and responsibilities
+### 2. API Bar Raiser
 
-Each RFC has a __driver__, __approver(s)__ and __stakeholders__.
+The public API of a feature represents the surface through which users interact
+with it, and we want to make sure these APIs are consistent, ergonomic and
+designed based on the intent and the mental model of our users. Additionally,
+once we announce that a feature is "stable" (1.0, GA, etc) any breaking change
+to its public API will require releasing a new major version, so we think of API
+decisions as "one way doors".
 
-The __driver__ is the person responsible to drive this RFC to completion. This
-does __not__ mean that the driver must be the person actually doing all the
-work, but it is the person responsible to get the RFC through the finish line.
+For each RFC, the core team will assign an **API Bar Raiser** who reviews and
+approves any public API of the feature. API Bar Raisers have veto rights on
+API-related design decisions, such as naming, structure, options, CLI commands
+and others.
 
-__Approvers__ are the people that have the authority to approve the RFC and any
-subsequent changes to it. Approvers have the final word if we cannot reach
-consensus.
+After an RFC is proposed, the core team will assign it an API Bar Raiser using a
+tiering model generally based on the size of the user base that will likely get
+exposed to the feature. As a general rule, the more "significant" the change is,
+we will assign a bar raiser with a wider and longer-term context of the project.
 
-__Stakeholders__ are people (internal/external) who may be impacted by this RFC
-or work on similar domains or projects. The goal is to make sure that
-stakeholders have an opportunity to influence the direction of the RFC in
-various stages of the process.
+API Bar Raisers are required to **sign off** on the **Working Backwards**
+section of your RFC, so we encourage to engage with them early in the process to
+make sure you are aligned on how the API should be designed.
 
-The driver is responsible to update the **Roles** table in the tracking issue.
+The API Bar Raiser is called out in the RFC tracking issue and in the RFC
+document itself. Feel free to reach us at the
```

Comment:
I'm confused about whether the bar raiser is assigned by the team or the author should reach out to get one assigned

Reply:
Either option is good. This is targeting both internal and external folks, so I wanted to give both options.